docs/features/dead_letter_archive.md

```diff
@@ -0,0 +1,19 @@
+---
+id: dead_letter_archive
+title: Cumulus Dead Letter Archive
+hide_title: false
+---
+
+This documentation explains the Cumulus dead letter archive and associated functionality.
+
+## DB Records DLQ Archive
+
+The Cumulus system contains a number of [dead letter queues](./dead_letter_queues). Of our system lambda functions, perhaps the most important of these is the `sfEventSqsToDbRecords` lambda function which parses Cumulus messages from workflow executions to generate and write database records to the Cumulus database.
```

Comment:
It doesn't look like `./dead_letter_queues` exists? Was this supposed to be `./lambda_dead_letter_queue`? I take it we need to use the filename instead of the doc ID in links (which is news to me).

Reply:
```suggestion
The Cumulus system contains a number of [dead letter queues](./lambda_dead_letter_queue) of our system lambda functions, perhaps the most important of these is the `sfEventSqsToDbRecords` lambda function which parses Cumulus messages from workflow executions to generate and write database records to the Cumulus database.
```

Reply:
Good catch! I totally went off that file's `id` instead of the file name. I'm just going to rewrite this sentence though, it reads pretty poorly.